### PR TITLE
Clean up for Langmuir tests

### PR DESCRIPTION
**Pull request by dpgrote**
The Langmuir README file was rewritten. It was very out of date, referring to load balancing tests that no longer existed.

File names were also cleaned up (renamed) as follows:
- input files:
  - `inputs_1d`
  - `inputs_2d`
  - `inputs_3d`
  - `inputs_rz`
- PICMI inputs:
  - `PICMI_inputs_2d.py`
  - `PICMI_inputs_3d.py`
  - `PICMI_inputs_rz.py`
- analysis scripts:
  - `analysis_1d.py`
  - `analysis_2d.py`
  - `analysis_3d.py`
  - `analysis_rz.py`

### CODE REVIEW

File: Examples/Tests/langmuir/README.md
Comment (EZoni) [inline Examples/Tests/langmuir/README.md:1]: Thank you for the PR, @dpgrote!

Maybe we could also take the chance to make the names among input files, PICMI inputs, and analysis scripts consistent.

For example, we could have:
- input files:
  - `inputs_1d`
  - `inputs_2d`
  - `inputs_3d`
  - `inputs_rz`
- PICMI inputs:
  - `PICMI_inputs_2d.py`
  - `PICMI_inputs_3d.py`
  - `PICMI_inputs_rz.py`
- Analysis scripts:
  - `analysis_1d.py`
  - `analysis_2d.py`
  - `analysis_3d.py`
  - `analysis_rz.py`
  
I could push these changes, too, if that is easier. Please let me know. It is also fine if you want to keep the PR smaller and leave it as it is. The cleanup of the README.md is necessary anyways and it looks good to me.

Comment (dpgrote) [reply Examples/Tests/langmuir/README.md:1]: @EZoni That sounds like a good idea. Go ahead and push these name changes it you have the chance.

Comment (EZoni) [reply Examples/Tests/langmuir/README.md:1]: Done in 0cb23b7.

**Review by EZoni — APPROVED** [review]
Thank you for cleaning this up, @dpgrote!

**Pull request merged**